src/types/patch.ts

```diff
@@ -11,7 +11,8 @@ export enum PatchStatus {
   Created = "created",
   Failed = "failed",
   Started = "started",
-  Success = "succeeded",
+  LegacySucceeded = "succeeded",
+  Success = "success",
```

Comment:
this might be more of a question for annie rather than for you haha but just want to double check that `"success"` is expected to be the new status? EVG-20032 says that we are going to use `"succeeded"` for both patches and versions but based on this PR it sounds like it's the opposite? 🤔

Reply:
Good point, @ablack12 `success` is the new status, correct? Should EVG-20032 be updated?

Reply:
Ah yeah that ticket description is old, I'll update it.